Tasks/Maven/CodeAnalysis/Common/FindbugsTool.ts

```diff
@@ -71,25 +72,20 @@ export class FindbugsTool extends BaseTool {
      * @returns a tuple of [affected_file_count, violation_count]
      */
     protected parseXmlReport(xmlReport: string, moduleName: string): [number, number] {
-        let fileCount = 0;
-        let violationCount = 0;
+        var jsonCounts:[number, number] = [0, 0];
 
         var reportContent = fs.readFileSync(xmlReport, 'utf-8');
         xml2js.parseString(reportContent, (err, data) => {
             // If the file is not XML, or is not from FindBugs, return immediately
             if (!data || !data.BugCollection ||
                 !data.BugCollection.FindBugsSummary || !data.BugCollection.FindBugsSummary[0] ||
-                !data.BugCollection.FindBugsSummary[0].FileStats) {
+                !(data.BugCollection.FindBugsSummary[0].FileStats || data.BugCollection.FindBugsSummary[0].PackageStats)) {
                 tl.debug(`[CA] Empty or unrecognized FindBugs XML report ${xmlReport}`);
                 return null;
             }
 
-            data.BugCollection.FindBugsSummary[0].FileStats.forEach((file: any) => {
```

Comment:
Since FindBugs' XML output is far more complex than that of PMD or Checkstyle, I opted to move it out into a seperate function.